include/wlr/types/wlr_input_method_v2.h

```diff
@@ -41,16 +43,29 @@ struct wlr_input_method_v2 {
 	bool client_active; // state known to the client
 	uint32_t current_serial; // received in last commit call
 
+	struct wlr_input_method_keyboard_grab_v2 *im_keyboard_grab;
```

Comment:
Can rename to just `keyboard_grab`, since it's already a member of `struct wlr_input_method_v2` it's unambiguous.

Reply:
I have renamed all im_keyboard_grab* to keyboard_grab*.